src/Physics/HadronTensors/HadronTensorI.h

```diff
@@ -42,6 +42,8 @@ typedef enum HadronTensorType {
   kHT_MEC_DeltaAll,
   kHT_MEC_Deltapn,
   kHT_MEC_EM,
+  kHT_MEC_EM_pn,
```

Comment:
Is the HadronTensorType ever saved to any output?   If it is then it's generally not a good idea to add new enum's in the middle of the list.   If it's only ever used internally to the code then it's probably fine-ish.

Reply:
Good catch, @nusense! Yeah, it's only used as an internal representation, so I think we can just shrug and move on. But I agree with the "best practices" advice wholeheartedly.